### PR TITLE
Mostly refactorings.

### PR DESCRIPTION
**Pull request by vinjana**

### CODE REVIEW

File: resources/analysisTools/copyNumberEstimationWorkflow/conda.sh [hunk 1]
```diff
@@ -1,6 +0,0 @@
-#!/usr/bin/env bash
```
Comment (dankwart-de) [inline resources/analysisTools/copyNumberEstimationWorkflow/conda.sh]: Why is this file gone?

File: src/de/dkfz/b080/co/aceseq/ACESeqMethods.groovy [hunk 2]
```diff
@@ -113,7 +113,7 @@ public final class ACESeqMethods {
     public static Tuple2<TextFile, TextFile> mergeDelly(TextFile knownSegmentsFile) {
         TextFile svFile = (TextFile) BaseFile.constructManual(TextFile.class, knownSegmentsFile, null, null, null, null, "dellyFileTag", null, null);
         svFile.setAsSourceFile();
-        JobResult result = new JobResult(knownSegmentsFile.getExecutionContext(), null, JobDependencyID.getFileExistedFakeJob(knownSegmentsFile.getExecutionContext()), false, null, null, null);
+        JobResult result = new JobResult(new BEJobResult(knownSegmentsFile.getExecutionContext(), null, new FakeBEJob(new BEFakeJobID(BEFakeJobID.FakeJobReason.FILE_EXISTED)), null, null, null, null))
```
Comment (dankwart-de) [inline src/de/dkfz/b080/co/aceseq/ACESeqMethods.groovy]: Don't get it. BEJobResult is in the BE core right? It should not take the context as an argument! Why is this here?

File: resources/analysisTools/copyNumberEstimationWorkflow/conda.sh (hunk 1 above)
Comment (vinjana) [reply resources/analysisTools/copyNumberEstimationWorkflow/conda.sh]: It was moved but the original somehow got stuck here. The new position is together with the other environment scripts in "copyNumberEstimationWorkflows/environments/" subdirectory.

File: src/de/dkfz/b080/co/aceseq/ACESeqMethods.groovy (hunk 2 above)
Comment (vinjana) [reply src/de/dkfz/b080/co/aceseq/ACESeqMethods.groovy]: You are right. The value is actually dropped and another constructor is used. I remove the constructor with the first "Object" type parameter and fix this position.

Comment (vinjana) [reply src/de/dkfz/b080/co/aceseq/ACESeqMethods.groovy]: Actually, we could go even further. RoddyCore JobResult is now not more than a call to super() in the constructor. We could completely get rid of it.

Comment (vinjana) [reply src/de/dkfz/b080/co/aceseq/ACESeqMethods.groovy]: O.k. JobResult is deleted (see #2).

**Comment by vinjana** [comment]
I did a merge with develop and test now. When everything works I am going to pull #1 and #2.

**Comment by vinjana** [comment]
Runs on PBS with bioconda.
